DAOS-18231 test: Move tests dfuse/daos_build_vm.py to daos_build.py

shimizukko opened the pull request:
Move the tests from VM to HW Medium based on the
triage meeting. Remove daos_build_vm.py and yaml.

Skip-unit-tests: true
Skip-fault-injection-test: true
Skip-func-hw-test-medium: false
Test-tag: test_dfuse_daos_build_wt_il test_dfuse_daos_build_wt_pil4dfs

### Steps for the author:

* [ ] Commit message follows the [guidelines](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments).
* [ ] Appropriate [Features or Test-tag](https://daosio.atlassian.net/wiki/spaces/DC/pages/10984259629/Test+Tags) pragmas were used.
* [ ] Appropriate [Functional Test Stages](https://daosio.atlassian.net/wiki/spaces/DC/pages/12147556353/CI+Functional+Test+Stages) were run.
* [ ] At least two positive code reviews including at least one code owner from each category referenced in the PR.
* [ ] Testing is complete. If necessary, forced-landing label added and a reason added in a comment.

#### After all prior steps are complete:
* [ ] Gatekeeper requested (daos-gatekeeper added as a reviewer).

shimizukko committed:
DAOS-18231 test: Move tests dfuse/daos_build_vm.py to daos_build.py

Move the tests from VM to HW Medium based on the
triage meeting. Remove daos_build_vm.py and yaml.

Skip-unit-tests: true
Skip-fault-injection-test: true
Skip-func-hw-test-medium: false
Test-tag: test_dfuse_daos_build_wt_il test_dfuse_daos_build_wt_pil4dfs
Signed-off-by: Makito Kano <makito.kano@hpe.com>

daosbuild3 commented:
Test stage Functional Hardware Medium completed with status FAILURE.  https://jenkins-3.daos.hpc.amslabs.hpecorp.net//job/daos-stack/job/daos/view/change-requests/job/PR-17272/1/execution/node/828/log

shimizukko committed:
DAOS-18231 test: Update comment

Skip-unit-tests: true
Skip-fault-injection-test: true
Skip-func-hw-test-medium: false
Test-tag: test_dfuse_daos_build_wt_il test_dfuse_daos_build_wt_pil4dfs
Signed-off-by: Makito Kano <makito.kano@hpe.com>

shimizukko commented:
Couldn't access hdr-137. Try running the tests again.

shimizukko committed:
DAOS-18231 test: Remove run_on_vms=True

Skip-unit-tests: true
Skip-fault-injection-test: true
Skip-func-hw-test-medium: false
Test-tag: test_dfuse_daos_build_wt_il test_dfuse_daos_build_wt_pil4dfs
Signed-off-by: Makito Kano <makito.kano@hpe.com>

shimizukko commented:
Build failure on both PMEM and MD-on-SSD cluster.
```
Timeout building over dfuse with il in mode writethrough
Error creating test-specific temporary directory on hdr-123
```

shimizukko committed:
Merge branch 'master' into makito/DAOS-18231

Skip-unit-tests: true
Skip-fault-injection-test: true
Skip-func-hw-test-medium: false
Test-tag: test_dfuse_daos_build_wt_il test_dfuse_daos_build_wt_pil4dfs

shimizukko committed:
DAOS-18231 test: Increase build_time to 90

This is the base value and it may be multiplied by 2 or 4
depending on the cache_mode. For example, cach_mode
writethrough timed out after 120 min, so it'll spend up
to 180 min now.

Skip-unit-tests: true
Skip-fault-injection-test: true
Skip-func-hw-test-medium: false
Test-tag: test_dfuse_daos_build_wt_il test_dfuse_daos_build_wt_pil4dfs
Signed-off-by: Makito Kano <makito.kano@hpe.com>

shimizukko committed:
DAOS-18231 test: Fix pylint

Skip-unit-tests: true
Skip-fault-injection-test: true
Skip-func-hw-test-medium: false
Test-tag: test_dfuse_daos_build_wt_il test_dfuse_daos_build_wt_pil4dfs
Signed-off-by: Makito Kano <makito.kano@hpe.com>

daosbuild3 commented:
Test stage Test RPMs on EL 8.6 completed with status FAILURE.  https://jenkins-3.daos.hpc.amslabs.hpecorp.net/job/daos-stack/job/daos/job/PR-17272/4/display/redirect

shimizukko committed:
Merge branch 'master' into makito/DAOS-18231

Skip-unit-tests: true
Skip-fault-injection-test: true
Skip-func-hw-test-medium: false
Test-tag: test_dfuse_daos_build_wt_il test_dfuse_daos_build_wt_pil4dfs

shimizukko committed:
DAOS-18231 test: Increase build_time to 120

Skip-unit-tests: true
Skip-fault-injection-test: true
Skip-func-hw-test-medium: false
Test-tag: test_dfuse_daos_build_wt_il test_dfuse_daos_build_wt_pil4dfs
Signed-off-by: Makito Kano <makito.kano@hpe.com>

shimizukko committed:
Merge branch 'master' into makito/DAOS-18231

Skip-unit-tests: true
Skip-fault-injection-test: true
Skip-func-hw-test-medium: false
Test-tag: test_dfuse_daos_build_wt_il test_dfuse_daos_build_wt_pil4dfs

shimizukko committed:
Merge branch 'master' into makito/DAOS-18231

Skip-unit-tests: true
Skip-fault-injection-test: true
Skip-func-hw-test-medium: false
Test-tag: test_dfuse_daos_build_wt_il test_dfuse_daos_build_wt_pil4dfs

shimizukko committed:
DAOS-18231 test: Increase build_time to 240

Skip-unit-tests: true
Skip-fault-injection-test: true
Skip-func-hw-test-medium: false
Test-tag: test_dfuse_daos_build_wt_il test_dfuse_daos_build_wt_pil4dfs
Signed-off-by: Makito Kano <makito.kano@hpe.com>

daosbuild3 commented:
Test stage Functional Hardware Medium MD on SSD completed with status FAILURE.  https://jenkins-3.daos.hpc.amslabs.hpecorp.net//job/daos-stack/job/daos/view/change-requests/job/PR-17272/8/execution/node/831/log
